kb/deadline-exceeded-troubleshooting.md

```diff
@@ -0,0 +1,104 @@
+---
+slug: deadline-exceeded-troubleshooting
+title: DeadlineExceeded error troubleshooting
+tags:
+  - kb-article
+date: 2022-11-08T00:00:00Z
+---
+
+All requests made to the [Temporal Cluster](/concepts/what-is-a-temporal-cluster) are [gRPC requests](https://grpc.io/docs/what-is-grpc/core-concepts/#deadlines).
+Sometimes, when these requests can't be completed, you'll see this particular error message: `Context: deadline exceeded`.
+
+`Context: deadline exceeded` is an error that occurs when requests are not completed on time.
+Network interruptions, short timeouts, server overload, and Query errors can cause this error.
+
+The following sections will discuss the nature of this error and how to troubleshoot it.
+
+#### Check your logs
+
+:::note
+
+Cloud users cannot access some of the logs needed to diagnose the source of the error.
```

Comment:
Maybe something along the lines of "For Cloud users,  check your Workflow Execution history for errors, look for latencies and errors in your Cloud metrics, and create a support ticket with Temporal with details of the error you see and any additional information you can provide." ?
@tsurdilo would this be useful?

Reply:
I think just telling users to "look for latencies" is probably very specific. Maybe omit that part and we an just say if users sees this error to open a support ticket with full message